pkg/services/ngalert/api/api_convert_prometheus.go

```diff
@@ -28,9 +59,128 @@ func (srv *ConvertPrometheusSrv) RouteConvertPrometheusGetNamespace(c *contextmo
 }
 
 func (srv *ConvertPrometheusSrv) RouteConvertPrometheusGetRuleGroup(c *contextmodel.ReqContext, namespaceTitle string, group string) response.Response {
-	return response.Error(501, "Not implemented", nil)
+	// Just to make the mimirtool rules load work. It first checks if the group exists, and if the endpoint returns 501 it fails.
+	return response.YAML(http.StatusOK, apimodels.PrometheusRuleGroup{})
 }
 
-func (srv *ConvertPrometheusSrv) RouteConvertPrometheusPostRuleGroup(c *contextmodel.ReqContext, namespaceTitle string, prometheusGroup apimodels.PrometheusRuleGroup) response.Response {
-	return response.Error(501, "Not implemented", nil)
+func (srv *ConvertPrometheusSrv) RouteConvertPrometheusPostRuleGroup(c *contextmodel.ReqContext, namespaceTitle string, promGroup apimodels.PrometheusRuleGroup) response.Response {
+	logger := srv.logger.FromContext(c.Req.Context())
+
+	logger.Info("Converting Prometheus rule group", "folder_title", namespaceTitle, "group", promGroup.Name, "rules", len(promGroup.Rules))
+
+	ns, errResp := srv.getOrCreateNamespace(c, namespaceTitle, logger)
+	if errResp != nil {
+		return errResp
+	}
+
+	logger.Debug("Found folder", "folder_title", ns.Title, "folder_uid", ns.UID)
```

Comment:
it's a little bit deceiving and may mean that folder already existed. Perhaps change wording?